apps/condo/domains/common/components/GraphQlSearchInput.tsx

```diff
@@ -53,15 +58,17 @@ export const GraphQlSearchInput: React.FC<ISearchInputProps> = (props) => {
         searchAgainDependencies = [],
         initialValue,
         getInitialValueQuery,
+        infinityScroll,
         ...restProps
     } = props
     const client = useApolloClient()
     const [selected, setSelected] = useState('')
     const [isLoading, setLoading] = useState(false)
     const [data, setData] = useState([])
+    const [isAllData, setIsAllData] = useState(false)
```

Comment:
If this flag have meaning like "when all data has been loaded to list" may be better to name it like `isAllDataLoaded`?